lte/gateway/python/magma/mobilityd/tests/uplink_gw_tests.py

```diff
@@ -0,0 +1,43 @@
+"""
```

Comment:
It's a good practice to name py tests with `test` prefix, so this would be better named as `test_uplink_gw.py`

Reply:
I agree, but it is not followed by existing test, so I did not do it.